snippets/powershell/Post-NB-Cleanup-Assets.ps1

```diff
@@ -0,0 +1,168 @@
+<#
+
+.SYNOPSIS
+This sample script demonstrates the use of NetBackup REST API for listing all assets.
```

Comment:
Script appars to be listing as well as cleaning up the assets.

Reply:
Thanks for pointing out. I edited those comments to reflect the script properly.